inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
The reason why one needs to wait for `webcoll` to run in legacy is because otherwise the user will receive an email saying: You can visit the record in `http://inspirehep.net/record/123` and when clicking the link the record will not be available.

Only after `webcoll` has run the record is really accessible to the user.